TUM Campus App/CardViewController.swift

```diff
@@ -113,12 +103,36 @@ extension CardViewController {
         self.navigationItem.titleView = view
     }
     
-    func setupTableView() {
-        refresh.addTarget(self, action: #selector(CardViewController.refresh(_:)), for: UIControlEvents.valueChanged)
-        tableView.addSubview(refresh)
+    @objc func refresh(sender: AnyObject?) {
+        composedDataSource?.refresh()
+//        TODO fix this
```

Comment:
Would be great, to not commit commented code / todos.

Reply:
Its a work in progress :D